tileserver/__init__.py

```diff
@@ -234,9 +234,10 @@ def async_store(store, tile_data, coord, format):
 def async_update_tiles_of_interest(redis_cache_index, coord):
     """update tiles of interest set"""
     try:
-        redis_cache_index.index_coord(coord)
+        if coord.zoom <= 20:
+            redis_cache_index.index_coord(coord)
         if coord.zoom > 18:
-            coord_at_z18 = coord.zoomTo(18)
+            coord_at_z18 = coord.zoomTo(18).container()
```

Comment:
I think so. If a user navigated to tiles at z19 but not z18, we would never reprocess the z19 tile if it didn't exist at z18. We generate an expiry list from osm2pgsql at z18, and we intersect that list (and their parents) with our tiles of interest set. If it's not in the tiles of interest, it wouldn't get processed.